dali/base/dafdesc.cpp

```diff
@@ -398,7 +404,23 @@ struct CClusterInfo: implements IClusterInfo, public CInterface
                 name.set(gname);
         }
     }
-
+    void checkStriped()
+    {
+        if (!name.isEmpty())
+        {
+            Owned<IStoragePlane> plane = getDataStoragePlane(name, false);
+#ifdef _CONTAINERIZED
+            mspec.numStripedDevices = plane ? plane->numDevices() : 1;
+            if (mspec.numStripedDevices>1)
+                mspec.flags |= CPDMSF_striped;
+            else
+                mspec.flags &= ~CPDMSF_striped;
+#else
+            // Bare-metal can have multiple devices per plane (e.g. data + mirror), but it doesn't stripe across them
+            mspec.numStripedDevices = 1;
```

Comment:
That is possibly a flaw.  May want to revisit bare metal at some point.

Reply:
agreed, it would break bare metal to consider what it thinks of multidevice planes now as striped, but we could have a switch where new plane types were flagged as striped.
As you say, for future.